packages/react-dom/src/__tests__/ReactDOMFizzServerBrowser-test.js

```diff
@@ -272,22 +266,14 @@ describe('ReactDOMFizzServerBrowser', () => {
     } catch (error) {
       caughtError = error;
     }
-    expect(caughtError.message).toBe(
-      'The render was aborted by the server without a reason.',
-    );
-    expect(errors).toEqual([
-      'The render was aborted by the server without a reason.',
-    ]);
+    expect(caughtError.message).toBe('The operation was aborted.');
```

Comment:
Might be worth a follow-up. Currently Fizz thinks `controller.abort()` means that `signal.reason === undefined`. But the reason is only undefined if we didn't abort yet. If we did abort, reason will either be the value given in `abort` or a DOMException if reason is `undefined` (see https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/reason)

`controller.abort()` i.e. "no reason"
```bash
let controller = new AbortController()
undefined
controller.signal.reason
undefined
controller.abort()
undefined
controller.signal.reason
DOMException: signal is aborted without reason
    at <anonymous>:1:12
controller.signal.reason.message
'signal is aborted without reason'
```

`controller.abort('reason')`:
```bash
controller = new AbortController()
AbortController {signal: AbortSignal}
controller.signal.reason
undefined
controller.abort('the reason')
undefined
stats.ts:42          POST https://api.github.com/_private/browser/stats net::ERR_BLOCKED_BY_CLIENT
j @ stats.ts:42
requestIdleCallback (async)
i @ stats.ts:24
await in i (async)
g @ stats.ts:15
(anonymous) @ report-long-tasks.ts:6
controller.signal.reason
'the reason'
```